python/snewpy/snowglobes.py

```diff
@@ -96,13 +96,6 @@ def generate_time_series(model_path, model_type, transformation_type, d, output_
 
     # now process log data
     if (log_bins==True):
```

Comment:
Python lets you simply write `if log_bins:`, which is cleaner and easier to read.